account-settings/set_user_preferences.py

```diff
@@ -4,7 +4,6 @@
 # -------------------------------------------------------------------------
 # Mattermost server
 url = "http://35.153.176.26:8065"
-team_name = "demoteam"
 login_user = "admin@mattermost.com"
```

Comment:
Should we prompt for these values as well?

Reply:
Sounds good. I was thinking about "last_activity_at" to determine newer users :)